### DISCUSSION TITLE
Schedule example for Battery soc updating more often

### DISCUSSION THREAD

**Discussion by f1wade** [kellerza/sunsynk#241] [Q&A] [answered]
I know I'm late to the schedules, https://kellerza.github.io/sunsynk/reference/schedules
but I have just updated to these, and want to ensure my battery soc is set to update every change.
but as its a % I wasn't sure what 'KEY' to use.

I have home assistant sunsynk config containing:
SENSORS:
  - battery_soc
  
  SCHEDULES:
  - KEY: W
    READ_EVERY: 5
    REPORT_EVERY: 60
    CHANGE_ANY: false
    CHANGE_BY: 80
    CHANGE_PERCENT: 0
  - KEY: "%"
    READ_EVERY: 30
    REPORT_EVERY: 60
    CHANGE_ANY: true

I added the "%" as it didn't like %.

but think it may be wrong, so any help appritiated.

**Comment by f1wade** [comment, marked as answer]
Just noting that so far this seems to be working as I want. so is likely correct for others if they need it.

**Comment by kellerza** [comment]
During startup the log will print a list of schedules. % should be a valid unit

Feel free to add a % example to the docs

https://kellerza.github.io/sunsynk/reference/schedules

**Reply by f1wade** [reply]
Thanks for adding schedules, I been meaning to try these to fix my sometimes over export or over import for weeks.
I finally updated and added this and it's working great.

**Reply by f1wade** [reply]
I have edited, and created pull requests.

**Comment by RhysMcW** [comment]
you can specify a sensor for the schedule so for battery SoC something like this
- KEY: battery_soc
  READ_EVERY: 30
  REPORT_EVERY: 60
  CHANGE_ANY: true
  CHANGE_BY: 1
  CHANGE_PERCENT: 1
